test/Microsoft.AspNet.Razor.Test/CodeGenerators/CSharpTagHelperRenderingTest.cs

```diff
@@ -404,7 +404,7 @@ public static TheoryData DesignTimeTagHelperTestData
             get
             {
                 // Test resource name, baseline resource name, expected TagHelperDescriptors, expected LineMappings
-                return new TheoryData<string, string, IEnumerable<TagHelperDescriptor>, List<LineMapping>>
+                return new TheoryData<string, string, IEnumerable<TagHelperDescriptor>, IList<LineMapping>>
```

Comment:
Realize the Razor data sets are inconsistent between lists and arrays. But why introduce an inconsistency within one data set? Just use another `List<LineMapping>`. If we decide to make everything consistent one way or another, we can do that in one pass.